csi.proto

```diff
@@ -871,6 +883,46 @@ message ListVolumesResponse {
   // An empty string is equal to an unspecified field value.
   string next_token = 2;
 }
+message ControllerGetVolumeRequest {
+  option (alpha_message) = true;
+
+  // Identity information for a specific volume. This field is
+  // REQUIRED. ControllerGetVolume will return with current
+  // volume information.
+  string volume_id = 1;
+}
+
+message ControllerGetVolumeResponse {
+  option (alpha_message) = true;
+
+  message VolumeStatus{
+    // A list of all `node_id` of nodes that the volume in this entry
+    // is controller published on.
+    // This field is OPTIONAL. If it is not specified and the SP has
+    // the PUBLISH_UNPUBLISH_VOLUME controller capability, the CO
+    // MAY assume the volume is not controller published to any nodes.
+    // If the field is not specified and the SP does not have the
+    // PUBLISH_UNPUBLISH_VOLUME controller capability, the CO MUST
+    // not interpret this field.
+    // published_node_ids MAY include nodes not published to or
+    // reported by the SP. The CO MUST be resilient to that.
+    repeated string published_node_ids = 1;
+
+    // volume_condition shows information about the current health of
+    // the volume.
+    // This field is OPTIONAL.
+    // This field MUST be specified if the
+    // GET_VOLUME_VOLUME_CONDITION controller capability is supported.
+    VolumeCondition volume_condition = 2;
+  }
+
+  // This field is REQUIRED
+  Volume volume = 1;
+
+  // This field is OPTIONAL. This field MUST be specified if the
+  // GET_VOLUME_VOLUME_CONDITION controller capability is supported.
```

Comment:
can you update this? `GET_VOLUME_VOLUME_CONDITION` is no longer there.

Reply:
Fixed